github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```
@@ -31,6 +31,35 @@ presubmits:
           resources:
             requests:
               memory: "8Gi"
+  - name: pull-cdi-unit-test-s390x
+    skip_branches:
+      - release-\d+\.\d+
```

```suggestion
      - release-v\d+\.\d+
```

I think the CDI branches include the `v`

---

They do